gbpservice/neutron/tests/unit/nfp/configurator/api/v1/controllers/test_controller.py

```diff
@@ -10,18 +10,22 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-#import json
-import mock
-import unittest
 
+import mock
 import oslo_serialization.jsonutils as jsonutils
 import pecan
-from pecan import rest
+PECAN_CONFIG_FILE = '/opt/stack/gbp/gbpservice/nfp/configurator/api/config.py'
```

Comment:
Unit tests should run on any machine where code is downloaded. In this case, this path is not suitable right?